source/usage-examples/changeStream.txt

```diff
@@ -0,0 +1,57 @@
+=================
+Watch for Changes
+=================
+
+.. default-domain:: mongodb
+
+Open a Change Stream
+--------------------
+
+You can keep track of changes of data in MongoDB, such as changes to a
+collection, database, or deployment by opening a :manual:`Change
+Stream</changeStreams/>`. A ``Change Stream`` allows applications to
+watch for changes to data and react to those changes. You can open a
+``Change Stream`` by calling the :node-api:`watch()
+</Collection.html#watch>` method on a collection, database, or
+deployment. The ``watch()`` method takes an optional pipeline as the
+first parameter that is an array of stages to filter the :manual:`change
+events </reference/change-events/>` output. 
+
+The ``watch()`` method can take in additional options as the second
+parameter. Create an `Object
+<https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object>`_
+to specify additional options. Set ``fullDocument`` to
+``'updateLookup'`` to receive an event that contains the changes to the
+document as well as the entire altered document.
+
+Process the Change Stream Events
+--------------------------------
+
+The ``watch()`` command returns a ``Change Stream`` instance. Use the
+`EventEmitter.on()
+<https://nodejs.org/api/events.html#events_emitter_on_eventname_listener>`_.
+method on the ``Change Stream`` instance to add a listener function. Call
```

Comment:
Nitpick: it could be more clear to users new to these concepts to mention what the methods do before mentioning the names of them. Also, we should try to be consistent with "X command does Y" and "Use X for Y", at least within the same paragraph. Here's an example (combination of both suggestions):
You can capture events from a change stream instance by using a listener function. Call the watch() command to get a Change Stream instance. Add your listener function by calling the EventEmitter.on() method on that instance. Set the value of the first parameter to the string 'change' and add your callback function to the second parameter...

Reply:
Fixed - almost verbatim